assets/lokomotive-kubernetes/bare-metal/flatcar-linux/kubernetes/cl/worker.yaml.tmpl

```diff
@@ -92,7 +92,7 @@ storage:
           KUBELET_IMAGE_URL=docker://k8s.gcr.io/hyperkube
           KUBELET_IMAGE_TAG=v1.18.3
           KUBELET_IMAGE_ARGS="--exec=/usr/local/bin/kubelet"
-          NODE_LABELS="node.kubernetes.io/node"
+          NODE_LABELS="${kubelet_labels}"
```

Comment:
Why change it this way? On Packet, for example, we use:

```
          NODE_LABELS="node.kubernetes.io/node,${node_labels}"
```

Why not do the same?

Reply:
IMO the template should not make an assumption what labels will be used. This is aiming towards https://github.com/kinvolk/lokomotive/issues/449. The less data we hardcode in the template, the better also.

Reply:
There are other paths towards #449, code-duplication might be possible to remove with symlinks, for example.

I'd even argue this commit goes against #449: if the templates are all different, it's harder to remove duplication in the future. This commit is making templates across platforms unconsistentn and different.